src/app/cases/page.tsx

```diff
@@ -48,64 +57,67 @@ export default function Page() {
       <H1>Browse Available Cases</H1>
       <FiltersContainer>
         <FilterDropdown
-          defaultValue="Remote/In Person"
-          options={['Remote/In Person', 'Remote Only', 'In Person Only']}
-          onChange={v =>
-            setCaseFilters({ ...caseFilters, remote: v as string })
-          }
+          defaultValue={defaultFilterValues.remote}
```

Comment:
question for @davidqing6432: personally, I'm not a huge fan of the fact that both options being selected shows up as `Remote + 1 more` or `In Person + 1 more` depending on which one was selected first...

some alternatives I can think of:
- handle this one filter differently by making it so that if both options are selected, it just says `Remote/In Person` (don't love this because selecting everything has the same effect as selecting nothing, but looks different because the filter is filled in/blue)
- make the filter single-select?? so you'd only filter if you want to restrict options to one of the two?

Reply:
another alternative is to make it so it's sorted (i.e. always `Remote +1 more` when both are chosen, regardless of the order). do you think that could be better?

Reply:
definitely think the sorted approach would also improve things! we should prob discuss all of this at worksesh (it's on the agenda)

Reply:
i agree with the sorted approach!